Students/Semester2/lessons/students/1_basics/0_readings/04_github_commands.md

```diff
@@ -9,15 +8,15 @@ git push = pushes your commits to the remote repository/branch (sends it to gith
  
 git clone = copies the repository to a specific folder on your computer; copy and paste the URL right after typing this command.
  
-git pull = pulls down the changes from the remote repository into the branch you're in (you can use git pull to access lessons)
+git pull = pulls down the changes from the remote repository into the branch you're in (you can use git pull to access lessons).
```

Comment:
lets have consistency with these - if its a bullet in structure, dont have it end in a period. if its a sentence, case it appropriately and put a period